integration/epochs/cluster_epoch_test.go

```diff
@@ -175,6 +180,9 @@ func (s *Suite) CreateVoterResource(address sdk.Address, nodeID flow.Identifier,
 	err = registerVoterTx.AddArgument(cadence.NewString(nodeID.String()))
 	require.NoError(s.T(), err)
 
+	err = registerVoterTx.AddArgument(cadence.NewString(nodeID.String()))
```

Comment:
This is adding the `nodeID` again - assuming it should be using the new staking key argument.
```suggestion
	err = registerVoterTx.AddArgument(cadence.NewString(publicStakingKey.EncodedHexRepresentation()))
```